pkgs/data/fonts/iwona/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchzip }:
+
+stdenv.mkDerivation rec {
+  name = "iwona-${version}";
+  version = "0_995";
+
+  src = fetchzip {
+    url = "jmn.pl/pliki/Iwona-otf-${version}.zip";
+    sha256 = "1wj5bxbxpz5a8p3rhw708cyjc0lgqji8g0iv6brmmbrrkpb3jq2s";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/opentype/
+    cp -v *.otf $out/share/fonts/opentype/
+  '';
+
+  outputHashAlgo = "sha256";
+  outputHashMode = "recursive";
+  outputHash = "1dcpn13bd31dw7ir0s722bv3nk136dy6qsab0kznjbzfqd7agswa";
+
+  meta = with stdenv.lib; {
+    description = "A two-element sans-serif typeface, created by Małgorzata Budyta";
+    homepage = http://jmn.pl/en/kurier-i-iwona/;
+    # "[...] GUST Font License (GFL), which is a free license, legally
```

Comment:
I don't think we need this particular comment above the license. If they say it's legally equivalent, I think we can use the lppl you are using.

Reply:
Somebody stumble over the difference, so I would keep the remark also there is technically no difference.